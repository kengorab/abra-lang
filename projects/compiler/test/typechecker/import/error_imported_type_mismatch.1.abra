import Foo from "./_exports"

enum Color { Purple }

val _ = Foo(color: Color.Purple)
