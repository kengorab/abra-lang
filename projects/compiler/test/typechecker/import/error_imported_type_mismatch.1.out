Error at %FILE_NAME%:5:25
Type mismatch for parameter 'color'
  |  val _ = Foo(color: Color.Purple)
                             ^
Expected: Color
but instead found: Color
